src/main/kotlin/no/nav/familie/tilbake/domain/Behandling.kt

```diff
@@ -42,11 +82,26 @@ enum class Behandlingsstatus(val kode: String) {
     UTREDES("UTRED")
 }
 
-enum class Behandlingstype(val kode: String) {
+enum class Behandlingstype(val kode: String, val behandlingssteg: List<Behandlingsstegstype>) {
 
-    TILBAKEKREVING("BT-007"),
-    REVURDERING_TILBAKEKREVING("BT-009"),
-    UDEFINERT("-")
+    TILBAKEKREVING("BT-007", listOf(Behandlingsstegstype.INNHENT_OPPLYSNINGER,
+                                    Behandlingsstegstype.VARSEL_OM_TILBAKEKREVING,
+                                    Behandlingsstegstype.MOTTA_KRAVGRUNNLAG_FRA_ØKONOMI,
+                                    Behandlingsstegstype.FAKTA_OM_VERGE,
+                                    Behandlingsstegstype.FAKTA_OM_FEILUTBETALING,
+                                    Behandlingsstegstype.VURDER_FORELDELSE,
+                                    Behandlingsstegstype.VURDER_TILBAKEKREVING,
+                                    Behandlingsstegstype.FORESLÅ_VEDTAK,
+                                    Behandlingsstegstype.FATTE_VEDTAK,
+                                    Behandlingsstegstype.IVERKSETT_VEDTAK)),
+    REVURDERING_TILBAKEKREVING("BT-009", listOf(Behandlingsstegstype.HENT_GRUNNLAG_FRA_ØKONOMI,
+                                                Behandlingsstegstype.FAKTA_OM_VERGE,
+                                                Behandlingsstegstype.FAKTA_OM_FEILUTBETALING,
+                                                Behandlingsstegstype.VURDER_FORELDELSE,
+                                                Behandlingsstegstype.VURDER_TILBAKEKREVING,
+                                                Behandlingsstegstype.FORESLÅ_VEDTAK,
+                                                Behandlingsstegstype.FATTE_VEDTAK,
+                                                Behandlingsstegstype.IVERKSETT_VEDTAK))
```

Comment:
BT-007 og BT-009 var noen behandlingType-koder vi var litt tvunget til å bruke ifm Foreldrepenger. Vi burde gå bort fra disse og benytte våre egne navn/koder for behandlingsType. Tror @haldersaikat hadde noen gode forslag til nye navn.